mkosi/config.py

```diff
@@ -4952,7 +4952,7 @@ def parse_config(
     *,
     resources: Path = Path("/"),
     only_sections: Sequence[str] = (),
-) -> tuple[Args, Optional[Config], tuple[Config, ...]]:
+) -> tuple[Args, Path, Optional[Config], tuple[Config, ...]]:
```

Comment:
At this point I wonder whether this function shouldn't instead return something structured instead and while I do see the logic of adding it in second place, adding it there is a weird bit of churn, fortunately we are the only users of this.